_sass/module/_contributors.scss

```diff
@@ -13,6 +29,49 @@ $breakpoint--s2: $grid--medium;
   tr {
     border-bottom: 1px $color-grey-4 solid;
   }
+
+  .arrow-container {
+    width: 10px;
+    height: 10px;
+    display: inline-block;
+  }
+
+  .down-arrow {
+    background-color: $color-grey-5;
+
+    .arrow-container {
+      background-image: url('./images/down-arrow.svg');
+      background-position: top left;
+      // width: 10px;
+      // height: 10px;
+      // display: inline-block;
```

Comment:
any need for these comments? please delete them :)

Reply:
Sloppy commit - my bad